collections/common/molecule/default/molecule.yml

```
@@ -0,0 +1,35 @@
+---
+
+
+
+dependency:
+  name: galaxy
+  options:
+    requirements-file: ../../require.yml
+
+driver:
+  name: docker
```

container driver instead of the docker drive
/
```suggestion
  name: container
```